src/System.Net.Security/tests/FunctionalTests/SslStreamCredentialCacheTest.cs

```diff
@@ -42,8 +41,20 @@ public class SslStreamCredentialCacheTest
 
                 await Task.WhenAll(tasks).TimeoutAfter(15 * 1000);
 
-                Assert.True(client.IsMutuallyAuthenticated);
-                Assert.True(server.IsMutuallyAuthenticated);
+                if (!PlatformDetection.IsWindows7 ||
+                    Capability.IsTrustedRootCertificateInstalled())
+                {
+                    // https://technet.microsoft.com/en-us/library/hh831771.aspx#BKMK_Changes2012R2
+                    // Starting with Windows 8, the "Management of trusted issuers for client authentication" has changed:
+                    // The behavior to send the Trusted Issuers List by default is off.
+                    //
+                    // In Windows 7 the Trusted Issuers List is sent within the Server Hello TLS record. This list is built
```

Comment:
Makes sense - please open a tracking issue. We can create a better `Capability` to achieve that.